build/ci.go

```diff
@@ -763,15 +763,25 @@ func doAndroidArchive(cmdline []string) {
 			gpg := exec.Command("gpg", "--import")
 			gpg.Stdin = bytes.NewReader(key)
 			build.MustRun(gpg)
+
+			keyID, err := build.PGPKeyID(string(key))
+
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			// Upload the artifacts to Sonatype and/or Maven Central
```

Comment:
With this change, `-signer` is required for `-upload`. Please remove the `if` around the variable to make that more clear.